inlang/source-code/sdk2/src/project/loadProject.ts

```diff
@@ -55,7 +55,9 @@ export async function loadProject(args: {
 		db,
 		plugins: {
 			get: () =>
-				structuredClone(reactiveState.plugins$.getValue()) as InlangPlugin[],
+				// TODO loadMessages - we cant use structuredClone since we have a function
+				// structuredClone(reactiveState.plugins$.getValue()) as InlangPlugin[],
+				reactiveState.plugins$.getValue() as InlangPlugin[],
```

Comment:
Good remark. I would use the Typescript `Readonly` type in the api definition to leave the plugins as mutable but consumers from mutating because Typescript would complain.